utils/lists.ts

```diff
@@ -272,6 +272,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Companions adoptable by owners of The Doge NFT ($DOG)",
     url: "https://etherscan.io/address/0x76e3dea18e33e61de15a7d17d9ea23dc6118e10f#writeContract",
   },
+  {
+    name: "Dungeons (for Adventures)",
+    description: "On-chain generative Rogue style Dungeons with loot.  Free to mint for Loot holders.",
```

Comment:
Can you please make the description shorter so it fits in one line, also please remove the trailing period.

Reply:
I've done that now.  Thanks